src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -235,6 +235,33 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
 		setHideLowBalance(!hideLowBalance);
 	};
 
+	const addToMetamask = async (address: string, symbol: string, token: any) => {
+		if (symbol.length >= 11) {
+			symbol = symbol.slice(4, symbol.length);
+		}
+		//@ts-ignore
+		const url = iconsCoin[token.coin];
+		const decimals = 18;
+		try {
+			// wasAdded is a boolean. Like any RPC method, an error may be thrown.
+			//@ts-ignore
```

Comment:
Please remove TS-ignore and fix underlying type issue

Reply:
done